src/commands/runtime/package/delete.js

```diff
@@ -41,6 +83,11 @@ PackageDelete.args = [
 PackageDelete.flags = {
   json: flags.boolean({
     description: 'output raw json'
+  }),
+  recursive: flags.boolean({
+    description: 'Deletes all associated actions',
```

Comment:
Deletes all associated actions, and rules & triggers associated with the actions